xml/mysql_banners.xml

```diff
@@ -0,0 +1,736 @@
+<?xml version="1.0"?>
+<fingerprints matches="mysql.banners">
+
+   <fingerprint pattern="^(\d\.\d{1,2}\.\h{1,3}(?:[.-]\d{1,2})?(?:[.-]\d{1})?)(?:-m\d{1,2})?(?:-rc)?(?:-alpha)?(?:-beta)?(?:-gamma)?(?:-?[Mm]ax)?(?:-rs)?(?:-modified)?(?:-debug)?(?:-log)?$">
```

Comment:
Perhaps, but isn't in our data set yet.  MariaDB does use 10.x.x and that is accounted for in the matches starting at line 568.  The 'MariaDB MariaDB on Debian Lenny' fingerprint is missing this.  

I can adjust the regex as you requested.  I keep falling back to my 'unknown service' mindset and forgetting that the assumption is that we already know its MySQL and just want to fingerprint it.